webpack.config.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const isProduction = process.env.NODE_ENV == 'production';
+
+const config = {
+  entry: './src/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devServer: {
```

Comment:
Usually, it's better to split configs by separate files. Each file will contains its own configuration for different modes. I can see there that we can split this config at least on 3 files: prod, dev and common. PROD will contain at least optimisation config (excluding of source maps, optimisations for chunks, etc.), DEV will have like only dev server config, common - reusable item for both DEV and PROD. The last this is to merge these stuff, feel free to use [`webpack-merge`](https://www.npmjs.com/package/webpack-merge) library.